.github/workflows/electron-native-package.yml

```
@@ -0,0 +1,373 @@
+
+  # ---------------------------------------------------------------------------
+  # Windows ARM64: QHexRT (Hexagon NPU) only — llama.cpp/ONNX/Sherpa do not
+  # build on this target at all (ggml rejects MSVC for ARM; FetchONNXRuntime
+  # has no win-arm64 URL — see bindings/electron/AGENTS.md). Runs on the
+  # self-hosted runner registered on the real Snapdragon X2 Elite box, since
+  # this is the one lane no GitHub-hosted runner can do (real QAIRT/Hexagon
+  # toolchain + a receipt staged from the licensed QAIRT SDK).
+  # ---------------------------------------------------------------------------
+  native_win_arm64_qhexrt:
+    needs: validate
+    runs-on: [self-hosted, Windows, ARM64, qhexrt]
```

actionlint (1.7.12)

[error] label "qhexrt" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)